fastlane/Fastfile

```diff
@@ -59,6 +59,94 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release_info = get_latest_github_release(url: "RevenueCat/purchases-ios")
+  latest_android_release_info = get_latest_github_release(url: "RevenueCat/purchases-android")
+
+  latest_ios_release_name = latest_ios_release_info["name"]
```

Comment:
Do we foresee using other info from the latest release other than the name? If not, I suggest renaming `get_latest_github_release` to `get_latest_github_release_name` and return that directly in the action.